components/mailbox/src/Mailbox.svelte

```diff
@@ -317,8 +317,9 @@
       await dispatchDraft(event);
       return;
     }
-    dispatchEvent("threadClicked", { event, thread });
```

Comment:
This was removed because the threadClicked dispatched event bubbles up to composer from email, so it doesn't need to be dispatched from the mailbox component as well